### ISSUE TITLE
Show product name in Sale List

### ISSUE THREAD

**Issue by siusang** [uilibrary/Stocky-Issues-and-Feature-request#168]
Hello

In Sale List page, can i change the customer column to product? or make a new Product column. Now I need to click sales details every time in order to check which product was sold. thank you.

![qooah-screenshot_2801](https://user-images.githubusercontent.com/86476694/123549410-794e2800-d79b-11eb-8961-c5830370d2bf.jpg)

**Comment by hamzakifani** [comment]
we will discuss it and if this is reliable and useful for most of the users then we will add it to our queue.
if you like our work & support, you can leave a review that will motivate our team toward further development if possible. If you don’t know how to provide ratings & review please visit this link: https://help.market.envato.com/hc/en-us/articles/203269490-Giving-Feedback-On-Items thank you :)

**Comment by siusang** [comment]
Yes, please, this will make it easier for us to manage our sales, or if you can let us select which item to be shown in this chart will be perfect, thank you for all your team's hard work.

**Comment by siusang** [comment]
And also the Dashboard section, it is difficult to found out which product was sold recently. please make this change in coming updates, thank you so much, i love to using this app.

![qooah-screenshot_2802](https://user-images.githubusercontent.com/86476694/123597813-62084c80-d826-11eb-9751-2502b7744eb5.jpg)
